generators/init/index.js

```diff
@@ -5,21 +5,15 @@
 */
 const Generator = require('yeoman-generator');
 const chalk = require('chalk');
-const simpleFabricaConfigJson = require('../../samples/fabricaConfig-1org-1channel-1chaincode');
 
 module.exports = class extends Generator {
   async copySampleConfig() {
-    this.fs.writeJSON(
-      'fabrica-config.json',
-      simpleFabricaConfigJson,
-    );
+    this.fs.copy(this.templatePath(), this.destinationPath());
 
     this.on('end', () => {
       this.log('===========================================================');
       this.log(chalk.bold('Sample config file created! :)'));
-      this.log('');
-      this.log(`Chaincode directory is '${simpleFabricaConfigJson.chaincodes[0].directory}'.`);
-      this.log('If it\'s empty your network won\'t run entirely.');
+      this.log('You can start your network with \'fabrica up\' command');
```

Comment:
Well, overall propositions looks, but why using JS chaincode is an only option?  
IMO it's quite unfair for java chaincode as it is a valid and working option and it might be users choice (same with GO not added yet, but equally important).  

My idea/recommendation:  
- `fabrica init` : initializes fabrica-config.json and copies node chaincode (default option)  
- `fabrica init -l java`: does the same, but for java chaincode

Would suggest adding it in this PR or the next one (but really soon).

Reply:
will be resolved in #158